docs/customizations-api/javascript/api.md

```diff
@@ -607,6 +608,34 @@ window.lumapps.customize(({ components, constants }) => {
 | `className`     | CSS class that will be applied directly into the wrapper container.                   | No           | `string` or [css classes](#css-classes) | `undefined`   |
 | `hasBackground` | Whether or not the button has a background color.                                     | No           | `boolean`                               | `false`       |
 
+#### Card
+
+The `Card` component is a simple layout component used to display content in a frame with a default background and padding.
+
+![Component Card](./assets/component-card.png "Component Card")
+
+```js
+window.lumapps.customize(({ components, constants }) => {
+    const { Card } = components;
+    const { Kind } = constants;
+
+    const component = Card({
+        className: 'general-card',
+        children: 'Card content',
+        as: 'section',
+    });
+
+});
+```
+
+`Card` options:
+
+| Option          | Description                                                                           | Is required? | Option type                             | Default Value |
+|-----------------|---------------------------------------------------------------------------------------|--------------|-----------------------------------------|---------------|
+| `children`      | String that will be rendered inside the message.                                      | Yes          | `string`                                | `undefined`   |
```

Comment:
Here we should specify that it can be either a string or another component. It should also not say message 😅